fe/src/app/service/verify/components/InfoForm/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { object, string, array, number } from "yup";
+
+export const serviceInfoFormSchema = object({
+  serviceName: string().required(),
+  adress: string().required(),
+  city: string().required(),
+  weeklyWorkingHours: array()
+    .of(
+      object().shape({
+        dayOfWeek: number().min(0).max(6).required(),
+        startTime: string().required(),
+        endTime: string().required(),
+        lunchBreakStartTime: string().required(),
+        lunchBreakEndTime: string().required(),
+      })
+    )
+    .min(1)
+    .required(),
+  contactPhone: string().required(),
```

Comment:
Could add regix from our dotnet project to check if the number is correct Lithuanian number. But without message showing for this field, not sure if it is worth.